src/akamai/js/akamaiHD_flash.js

```diff
@@ -201,8 +203,30 @@ require("../../../html5-common/js/utils/constants.js");
      * @returns {boolean} True or false indicating success
      */
     this.setVideoUrl = function(url, encoding) {
-      /* Should be set to false if stream URL changes */
-      loaded = false;
+      var urlChanged = false;
+      controller = this.controller;
+
+      if (currentUrl.replace(/[\?&]_=[^&]+$/,'') != url)
+      {
+        currentUrl = url || "";
+
+        if (currentUrl.length > 0) 
+        {
+          currentUrl = currentUrl + (/\?/.test(currentUrl) ? "&" : "?") + "_=" + getRandomString();
+        }
+        urlChanged = true;
+        loaded = false;
+        url = "setVideoUrl("+currentUrl+")";
+      }
+      if (_.isEmpty(currentUrl)) 
+      {
+        controller.notify(controller.EVENTS.ERROR, { errorcode: 0 }); 
```

Comment:
Well VTC won't call load since you can't load an empty string and we don't want to destroy; we basically just want to be able to unset a url so that if the same url is loaded next it is treated as a fresh new video (for ads only right now but could expand).